src/Redistribution.sol

```diff
@@ -482,6 +486,21 @@ contract Redistribution is AccessControl, Pausable {
         require(false, "no matching commit or hash");
     }
 
+    /**
+     * @notice Returns minimum depth reveal has to have to participate in this round
+     */
+    function currentMinimumDepth() public view returns (uint8) {
+        // We are checking value in reveal phase, as the currentCommitRound is set to the current round
+        // but the currentClaimRound is still set to the last time claim was made
+        // We add 1 to ensure that for the next round the minimum depth is the same as last winner depth
+
+        uint8 skippedRounds = uint8(currentCommitRound - currentClaimRound) + 1;
+        uint8 lastWinnerDepth = winner.depth;
```

Comment:
can it be used in the first run as well? because then no claim happened to set a winner. is the declaration of the `winner` initializes all properties with def values?

Reply:
yes the first run is case that should be checked with testing for sure.
what should happen is that 
 winner.depth = 0
 currentClaimRound = 0
 currentCommitRound = 198902 or something so  it would be true on last condition and will be 0